.travis.yml

```diff
@@ -0,0 +1,24 @@
+language: ruby
+
+rvm:
+- 2.2
+- 2.1
+- 2.0.0
+- 1.9.3
+- 1.8.7
+- jruby-19mode
+- rbx-2
```

Comment:
Happy to keep it as one, but figured it would be good to ensure the database can be parsed/consumed by various versions to ensure compatibility. However, perhaps that's more a job for bundler-audit...